include/seqan3/argument_parser/detail/format_help.hpp

```diff
@@ -764,4 +765,84 @@ class format_version : public format_help
+     */
+    void parse(argument_parser_meta_data const & parser_meta)
+    {
+        std::ostream_iterator<char> out(std::cout);
+        meta = parser_meta;
+        std::stringstream seqan_license{
+        R"(Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Knut Reinert or the FU Berlin nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL KNUT REINERT OR THE FU BERLIN BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+DAMAGE.)"};
+
+        print_header();
+
+        if (!empty(meta.long_copyright))
+        {
+            std::cout << to_text("\\fB") << meta.app_name << " Copyright:\n";
+            std::fill_n(out, layout.leftPadding, ' ');
+            std::cout << to_text("\\fP") << meta.long_copyright << "\n\n";
+        }
+        else if (!empty(meta.short_copyright))
+        {
+            std::cout << to_text("\\fP") << meta.app_name
+                      << " full copyright information not available. Displaying"
+                      << " short copyright information instead.\n\n";
+            std::cout << to_text("\\fB") << meta.app_name << " Copyright:\n";
+            std::fill_n(out, layout.leftPadding, ' ');
+            std::cout << to_text("\\fP") << meta.short_copyright << "\n\n";
+        }
+        else
+            std::cout << to_text("\\fP") << meta.app_name << "copyright information not available.\n\n";
+
+        std::cout << to_text("\\fB") << "SeqAn3 Copyright:\n";
+        std::string line;
+        while (std::getline(seqan_license, line))
+        {
+            std::fill_n(out, layout.leftPadding, ' ');
```

Comment:
I am pretty sure that for this line and the one below you can just use the print_line function of the format
```cpp
print_line(line);
```
Maybe you need `print_line(line, false);`.

Reply:
I tried it, but with `print_line(line)` it treats every line as a paragraph and makes the license info huge. But with `print_line(line, false)` it treats the whole thing as one paragraph and removes the spacing in between... So maybe it's better to just keep it as is?